### ISSUE TITLE
Error when running afew: configparser.NoSectionError: No section: 'database'

### ISSUE THREAD

**Issue by jackkamm** [afewmail/afew#234]
When I try to run `afew --tag --new`, I get the following error:
```
Traceback (most recent call last):
  File "/home/jack/bin/afew", line 11, in <module>
    load_entry_point('afew==1.3.0', 'console_scripts', 'afew')()
  File "/home/jack/venv/afew/lib/python3.7/site-packages/afew/commands.py", line 142, in main
    with Database() as database:
  File "/home/jack/venv/afew/lib/python3.7/site-packages/afew/Database.py", line 20, in __init__
    self.db_path = notmuch_settings.get('database', 'path')
  File "/usr/lib/python3.7/configparser.py", line 780, in get
    d = self._unify_values(section, vars)
  File "/usr/lib/python3.7/configparser.py", line 1146, in _unify_values
    raise NoSectionError(section) from None
configparser.NoSectionError: No section: 'database'
```
This happens when I `pip install` in a venv, and when I install from ubuntu/arch package repos. The error goes away when I install with `python setup.py install --prefix=~/.local`. (But the install instructions suggest that `pip install` into a venv should work).

**Comment by GuillaumeSeren** [comment]
Hello,
If you want to 'afew --tag --new' you need to setup a config file, and this error seem's to come from that,
please check the doc (https://afew.readthedocs.io/en/latest/configuration.html) and give us more details on the setup.

**Comment by jackkamm** [comment]
I do have a config file, here is how it looks:
```
[SpamFilter]
[KillThreadsFilter]
[ListMailsFilter]
[ArchiveSentMailsFilter]
[Filter.1]
query = 'tag:killed'
tags = -unread
message = marking killed new email as read
[InboxFilter]
```
I am using Archlinux though previously I also tested on Ubuntu. The error occurs when I install from PyPi or from my system package manager (`pacman` on Arch, `apt` on Ubuntu). However, the error goes away when I install from source with `python setup.py install`. Please let me know if you need any more info.

**Comment by GuillaumeSeren** [comment]
Hey @jackkamm,
I think the database is located in the notmuch config (~/.notmuch-config) file itself, see
```sh
$ cat ~/.notmuch-config | grep databa
# Notmuch will store its database within a sub-directory of the path
[database]
```

**Comment by jackkamm** [comment]
I see, I had commented out the `[database]` section from `~/.notmuch-config` because I prefer the default location of `$HOME/mail`. Adding it back fixes the issue. I see now that support for the fallback `$HOME/mail` was added in 5c93af477939da070d4581e94131b0ec3ea319e4 but there has not been an official release since, which is why installing from source was working but not installing from PyPi or distro repos.
